packages/dashboard/src/components/querySettingsSync/utils.ts

```diff
@@ -0,0 +1,17 @@
+type SelectedOption = {
+  label?: string;
+  value?: string;
+};
+
+const SECOND_IN_MS = 1000;
+const MINUTE_IN_MS = 60000;
+export const DEFAULT_REFRESH_RATE = 5000; // 5 seconds
+export const DEFAULT_OPTION = { label: '5s', value: '5000' };
+
+export const refreshRateOptionMap = new Map<number, SelectedOption>([
```

Comment:
I would avoid using and exporting a raw `Map` object. It's an implementation detail which is hard to notice without visiting the file. You can see how the abstraction leaks in `refreshRateDropDown` as you're having to call specific `Map` APIs (i.e., `.get()` and `.values()`).

If you want to use a map, encapsulate it does not require the user to know it's a map. You can do that by creating an API like `findOption` and `getOptions`.